Client.xcodeproj/xcshareddata/xcschemes/Client.xcscheme

```diff
@@ -197,6 +197,16 @@
                ReferencedContainer = "container:Client.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
```

Comment:
When I open up the Client scheme it shows ClientUITests missing. Might just be a weird link in the scheme file.